llmfoundry/models/hf/hf_base.py

```diff
@@ -72,6 +72,7 @@ def __init__(
         additional_train_metrics: Optional[list] = None,
         additional_eval_metrics: Optional[list] = None,
         should_save_peft_only: bool = True,
+        peft_is_trainable: bool = False,
```

Comment:
do we need a config flag for this? shouldn't they just always be trainable?

Reply:
hm...yeah that's true. I put this flag there because I didn't want to change the default behavior, but the default behavior is to have `is_trainable=False` which we don't want in foundry anyways.

Modifying to reflect this